docs/management/admin/admin-pg-ov.asciidoc

```diff
@@ -1,77 +1,153 @@
 
 *Configuration status*
 
-The status of the configuration appears in the *Configuration Status* column and displays one of the following possibilities:
+The status of the Policy configuration appears in the *Policy Status* column and displays one of the following possibilities:
 
-* *Success:* The configuration applied successfully.
+* *Success:* The Policy applied successfully.
 
-* *Pending or Partially Applied:* The configuration is pending application, or the configuration in its entirety was not applied.
+* *Pending or Partially Applied:* The Policy is pending application, or the Policy was not applied in its entirety.
 
-NOTE: In some cases, some actions taken on the endpoint may fail during the configuration application but are not recognized as a critical failure - meaning there may be a failure, but the hosts are still protected. In this case, the configuration status will display as "Partially Applied."
+NOTE: In some cases, some actions taken on the endpoint may fail during the configuration application but are not recognized as a critical failure - meaning there may be a failure, but the endpoints are still protected. In this case, the configuration status will display as "Partially Applied."
 
-* *Failure:* The configuration did not apply correctly. As such, hosts are not protected.
+* *Failure:* The Policy did not apply correctly. As such, endpoints are not protected.
 
 * *Unknown:* The user interface is waiting for the API response to return, or, in rare cases, the API returns an undefined error or value.
 
-To view configuration status details, click the link and review the data in the flyout panel. In the following image, you can see that the `Agent Connectivity` failed, generating a "Failed" configuration status.
+To view Policy status details, click the link and review the data in the flyout panel. In the following image, you can see that the malware configuration and logging failed, generating a "Failed" Policy status.
 
 [role="screenshot"]
 image::images/config-status.png[Config status details]
 
 Expand each section and subsection to view individual responses from the agent.
 
-TIP: If you need help troubleshooting a configuration failure, see the {ingest-guide}/ingest-management-troubleshooting.html[Ingest Manager troubleshooting topic].
+TIP: If you need help troubleshooting a configuration failure, see the {ingest-guide}/ingest-management-troubleshooting.html[Fleet troubleshooting topic].
+
+*Filter endpoints*
+
+To filter the Endpoints list, use the Search bar to enter a query using *{kibana-ref}/kuery-query.html[{kib} Query Language (KQL)]*. To refresh the search results, click *Refresh*.
+
+[role="screenshot"]
+image::images/filter-endpoints.png[]
+
+NOTE: The timepicker on the right side of the page allows you to set a time interval to automatically refresh the Endpoints list -- for example, if new endpoints were added or deleted. It does not, however, update any filtered endpoints with new search results.
```

Comment:
We can delete the last line here -

> It does not, however, update any filtered endpoints with new search results.

If a user has filtered the endpoints list and new data comes in to match that filter, it will appear. Not sure if that's what was meant by this line but it could be confusing/misleading!